.github/workflows/gradle.yml

```diff
@@ -27,11 +27,12 @@ jobs:
         run: ./run-checks.sh
 
       - name: Validate Gradle Wrapper
-        uses: gradle/wrapper-validation-action@v1
+        uses: gradle/actions/wrapper-validation@v3
 
       - name: Setup JDK 17
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v4
         with:
+          distribution: 'zulu'
```

Comment:
For your information as to why Zulu has to be specified here for `setup-java@v4`:

When upgrading `actions/setup-java@v1` to `actions/setup-java@v4`, from `v2` onwards, the `distribution` parameter here is mandatory. Since the original CI workflow version (`v1`) did not specify a `distribution` parameter, it used the `zulu` distribution by default (https://github.com/actions/setup-java/blob/main/docs/switching-to-v2.md).

Reply:
Thanks for the additional info. Commit bodies have been updated and wrapped at 72 chars.